packages/video_player/video_player_web/lib/src/video_player.dart

```diff
@@ -122,6 +121,10 @@ class VideoPlayer {
       setBuffering(false);
       _eventController.add(VideoEvent(eventType: VideoEventType.completed));
     });
+
+    if (src != null) {
```

Comment:
Do you want to put a comment saying this should stay last (and why), so that people adding new code know to put it before this?

Reply:
Yes, good idea. I'll also comment that this is nullable for tests.